src/runtime/tests/recipe-test.ts

```diff
@@ -798,4 +798,21 @@ describe('recipe', () => {
     assert.equal(recipe.handles[1].ttl.units, TtlUnits.Minute);
     assert.isUndefined(recipe.handles[2].ttl);
   });
+  it('parses recipe handle capabilities', async () => {
+    const recipe = (await Manifest.parse(`
+      recipe Thing
+        h0: create persistent
+        h1: create tied-to-runtime 'my-id'
+        h2: create tied-to-arc #myTag
+        h3: create #otherTag`)).recipes[0];
+    const verifyRecipeHandleCapabilities = (recipe) => {
+      assert.lengthOf(recipe.handles, 4);
+      assert.deepEqual([...recipe.handles[0].capabilities], ['persistent']);
+      assert.deepEqual([...recipe.handles[1].capabilities], ['tied-to-runtime']);
+      assert.deepEqual([...recipe.handles[2].capabilities], ['tied-to-arc']);
```

Comment:
Can we add a second capability to one of these handles (and use `assert.sameMembers` to test their equality)?

I think this would help ensure that we don't introduce bugs in the implementation (e.g. assuming that there will only be one capability).

Reply:
done (i don't think this meaningful set of multiple values, but it doesn't matter in this test scope).